llvm/lib/Target/AMDGPU/AMDGPUIGroupLP.cpp

```diff
@@ -832,7 +832,8 @@ void PipelineSolver::solve() {
 enum IGLPStrategyID : int {
   MFMASmallGemmOptID = 0,
   MFMASmallGemmSingleWaveOptID = 1,
-  MFMAExpInterleave = 2
+  MFMAExpInterleave = 2,
+  MFMAExpSimpleInterleaveID = 3
```

Comment:
Do we document these strategies anywhere? Like what is the summary of the difference between `MFMAExpInterleave` and `MFMAExpSimpleInterleaveID`. Also it looks like ID suffix was forgotten for `MFMAExpInterleave` in a different patch?